config/config-test.ts

```diff
@@ -3,10 +3,11 @@ const isDev = process.env.NODE_ENV === 'development';
 const isLive = 'true';
```

Comment:
Line 1 in this file refers to chain id of Ropsten network. WDYT about changing it to 97 (BSC Testnet)?

Reply:
Testnet config doesn't work at all yet, should be fixed later alongside the reintroduction of the test version